/**:
  ros__parameters:
    camera_name: "camera_1"
    frame_id: "camera_top/camera_link"
    pixel_format: bgr8
    serial_no: 220300119
    resize_image: false
    camera_info_url: "file:///home/kaan/projects/arena_camera_leo_ws/src/arena_camera/config/camera0_info.yaml"
    # image settings
    fps: 15
    horizontal_binning: 4
    vertical_binning: 4
    exposure_auto: false
    exposure_target: 187
    gain_auto: false
    gain_target: 1
    gamma_target: 1.0


